macros/grants/grants.yml

```diff
@@ -0,0 +1,75 @@
+version: 2
+
+macros:
+  - name: grant_database_ownership_access
+    description: This macro grants ownership privilege to the specific roles within the database
```

Comment:
specific roles? or specified rolename

Reply:
This macro grants ownership privileges to the specified rolename on the database.

Reply:
done